### PR TITLE
[READY FOR REVIEW] Dan/populate Merkle tree state

### PR DESCRIPTION
**Pull request by DanTehrani**
## This PR
This PR completes a part of the task: [populating the database with data](https://github.com/personaelabs/nym/issues/7#issuecomment-1520520578).  This PR is scoped to populating the "Merkle tree state".  I will open a different PR for populating the "posts/upvotes" state.

## Description
Add a script to populate the database with (cached) address -> public key mapping, and multisig info. The cache will expedite the `writeTree` script, which creates and stores a Merkle tree of public keys in the database. 

- `writeTree` should be executed as a setup process in each of our local environments. 
- `writeTree` requires API keys for the Etherscan API and Alchemy, so I'll share my API keys as shared dev API keys in DMs.

### CODE REVIEW

File: packages/merkle_tree/src/writeTree.ts [hunk 1]
```diff
@@ -195,7 +199,10 @@ async function writeTree(blockHeight: number) {
       }
     }
   }
-  console.timeEnd("Get multisig guardians and pubkeys");
+  // Add the dev account
+  allAccounts.push(DEV_ACCOUNT);
```
Comment (cha0sg0d) [inline packages/merkle_tree/src/writeTree.ts:203]: @DanTehrani reminder to add a few more accounts so we can play with this more easily in a deployed DB.

**Review by cha0sg0d — APPROVED** [review]
Beyond adding a few dev accounts, this looks great! Excited for the populating posts / replies script too.

File: packages/merkle_tree/src/writeTree.ts (hunk 1 above)
Comment (DanTehrani) [reply packages/merkle_tree/src/writeTree.ts:203]: noted!

**Comment by DanTehrani** [comment]
@cha0sg0d 
Yes I will get the posts/replies state working ASAP!

Will merge this PR, thanks for the review!

**Pull request merged**